SpeechResponder/Help.md

```diff
@@ -134,6 +134,23 @@ Common usage of this is to provide further information about your rating, for ex
 
     You have been promoted {ExplorationRatingDetails("Surveyor").rank} times.
 
+### FactionDetails()
+
+This function will provide full information for a minor faction given its name.
+
+FactionDetails() typically takes a single argument of the faction name, but may add a system name for filtering.
+
+Common usage of this is to obtain a `Faction` object, providing current specifics of a minor faction, for example:
+
+    {set faction to FactionDetails("Lavigny's Legion")}
+	{if faction.name != "":
+	    {faction.name} is present in the
+	    {for presence in faction.presences:
+            {presence.systemName},
+        }
+		{if len(faction.presences) = 1: system |else: systems}.
+    }		    
```

Comment:
Indentations look like they may be a little off. Consider editing indentations for easier readability?

Reply:
I'm not sure why it looks strange here but  I did check it in the `Help` window and it looks fine.